examples/user_guide/Param.ipynb

```diff
@@ -192,6 +192,22 @@
    "source": [
     "The parameterized and annotated ``view`` method could return any one of the types handled by the [Pane objects](./Panes.ipynb) panel provides, making it easy to link parameters and their associated widgets to a plot or other output. Parameterized classes can therefore be a very useful pattern for encapsulating a part of a computational workflow with an associated visualization, declaratively expressing the dependencies between the parameters and the computation.\n",
     "\n",
+    "`pn.panel` can be used to use only a subset of parameters. For example, the code below uses only `phase`  while maintaining `sine.frequency` at value `1`"
```

Comment:
```suggestion
    "By default, a Parameters pane will show all parameters with a `precedence` value above the value `pn.Param.display_threshold`, so you can use `precedence` to automatically hide parameters that are not meant to have widgets.  You can also explicitly choose which parameters should have widgets in a given pane, by passing a `parameters` argument.  For example, the code gives a `phase`  widget, while maintaining `sine.frequency` at the initial value of `1`"
```

Reply:
That looks more readable and less confusing

Reply:
Glad to hear it; please commit the suggestion if it's ok (I don't have permissions to push to your fork).